core/src/types.ts

```diff
@@ -128,7 +128,10 @@ export type TRestTransformer<
 
 export type TModelFieldsConfig<TModel> = {
   fields: Record<keyof TModel, Field>;
-  postBuild?: (model: TModel) => Partial<TModel>;
+  postBuild?: (
```

Comment:
This changes allows for some edge cases with compatibility between new and legacy data models.